R/objects.R

```diff
@@ -1601,10 +1721,10 @@ as.SingleCellExperiment.Seurat <- function(x, assay = NULL, ...) {
     stop("Please install SingleCellExperiment from Bioconductor before converting to a SingeCellExperiment object")
   }
   assay <- assay %||% DefaultAssay(object = x)
-  sce <- SingleCellExperiment::SingleCellExperiment(assays = list(
-    counts = GetAssayData(object = x, assay = assay, slot = "counts"),
+  cds <- monocle::newCellDataSet(
```

Comment:
Is there a reason why we're using `monocle::newCellDataSet` in `as.SingleCellExperiment`?